### PR TITLE
Refactor ContentStyle

### PR DESCRIPTION
**Pull request by mori-atsushi**

### CODE REVIEW

File: compose-stylesheet-core/src/commonMain/kotlin/com/moriatsushi/compose/stylesheet/content/ProvideContentStyle.kt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.moriatsushi.compose.stylesheet
+package com.moriatsushi.compose.stylesheet.content
```
Comment (PANDATD) [inline compose-stylesheet-core/src/commonMain/kotlin/com/moriatsushi/compose/stylesheet/content/ProvideContentStyle.kt:2]: hey there, please remove the unnecessary space

**Review by PANDATD — COMMENTED** [review]
just need to remove the unnecessary space.

File: compose-stylesheet-core/src/commonMain/kotlin/com/moriatsushi/compose/stylesheet/content/ProvideContentStyle.kt (hunk 1 above)
Comment (mori-atsushi) [reply compose-stylesheet-core/src/commonMain/kotlin/com/moriatsushi/compose/stylesheet/content/ProvideContentStyle.kt:2]: Sorry, where is the space you mentioned?

Comment (PANDATD) [reply compose-stylesheet-core/src/commonMain/kotlin/com/moriatsushi/compose/stylesheet/content/ProvideContentStyle.kt:2]: sorry, i apologized for the following

**Pull request merged**